webapp/src/store/Store.ts

```diff
@@ -1,11 +1,38 @@
+import path from 'path';
 import { ITranslationAdapter, TranslationMap } from '@/utils/adapters';
 import { LanguageNotFound, MessageNotFound } from '@/errors';
 
 type StoreData = {
   languages: TranslationMap;
 };
 
-export default class Store {
+export class Store {
```

Comment:
I think this new class should have been in a separate file from the old one.

Reply:
will do